src/app/content/components/BookBanner.tsx

```diff
@@ -0,0 +1,128 @@
+import Color from 'color';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import styled, { css } from 'styled-components';
+import { ChevronLeft } from 'styled-icons/boxicons-regular/ChevronLeft';
+import { maxNavWidth } from '../../components/NavBar';
+import { h3MobileLineHeight, h3Style, h4Style } from '../../components/Typography';
+import theme from '../../theme';
+import { AppState } from '../../types';
+import * as select from '../selectors';
+import { Book, Page } from '../types';
+import { bookDetailsUrl, findArchiveTreeSection } from '../utils';
+
+export const bookBannerDesktopHeight = 13;
+export const bookBannerMobileHeight = 10.4;
+
+// tslint:disable-next-line:variable-name
+const LeftArrow = styled(ChevronLeft)`
+  height: 2rem;
+  width: 2rem;
+  color: ${theme.color.neutral.base};
+  margin-right: 0.7rem;
+`;
+
+interface PropTypes {
+  page?: Page;
+  book?: Book;
+}
+
+// tslint:disable-next-line:variable-name
+const TopBar = styled.div`
+  width: 100%;
+  max-width: ${maxNavWidth}rem;
+  margin: 0 auto;
+`;
+
+const bookBannerTextStyle = css`
+  max-width: 87rem;
```

Comment:
Is this value based on another number (like the `@media (width)` breakpoint)? If so, could they be linked somehow in the code as a comment or by basing this value on the other?

Reply:
no this was intended to stop the text from going over the background image, but the background image got removed from the spec, so maybe we don't even need this anymore? i'll ask about it

Reply:
this is `1170 - 570 / 2` or `pageWidth - textWidth / 2`, i'll replace with the math